### PR TITLE
[release-4.4]Bug 1832999: Migrate to MAO

### PR DESCRIPTION
**Pull request by iamemilio**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	github.com/ajeddeloh/yaml v0.0.0-20170912190910-6b94386aeefd // indirect
+	github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d // indirect
+	github.com/coreos/container-linux-config-transpiler v0.9.0
+	github.com/coreos/go-systemd v0.0.0-20190620071333-e64a0ec8b42a // indirect
+	github.com/coreos/ignition v0.33.0 // indirect
+	github.com/golang/groupcache v0.0.0-20191027212112-611e8accdfc9 // indirect
+	github.com/gophercloud/gophercloud v0.6.1-0.20191025185032-6ad562af8c1f
+	github.com/gophercloud/utils v0.0.0-20190124231947-9c3b9f2457ef
+	github.com/openshift/client-go v0.0.0-20200320150128-a906f3d8e723
+	github.com/openshift/machine-api-operator v0.2.1-0.20200402110321-4f3602b96da3
```
Comment (michaelgugino) [inline go.mod:16]: Yes, this is pulling from the 4.5 branch.  We can't simply backport this kind of thing, need to do do the go get stuff separate from the application code.

**Comment by iamemilio** [comment]
https://github.com/openshift/cluster-api-provider-openstack/pull/98

**Pull request closed without merging**